stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/data/PredefinedUserCredentials.kt

```diff
@@ -26,6 +26,24 @@ object PredefinedUserCredentials {
     const val API_KEY: String = "qx5us2v6xvmh"
 
     val availableUsers: List<UserCredentials> = listOf(
+        UserCredentials(
```

Comment:
We should remove these before merging

Reply:
@kanat @TGazica can we just make this a patch and remove it from the PR so that we don't accidentally merge it